SECP-service/src/main/java/com/visucius/secp/DTO/UserDTO.java

```diff
@@ -10,6 +10,12 @@
     @JsonProperty
     private long userID;
 
+    @JsonProperty
+    private String displayName;
+
+    @JsonProperty
+    private String avatar_url;
```

Comment:
I have a question. So can a person upload a png file this way? If yes, then great. If not, then I don't know if urls might be the best way. So the way I have it right now (can change so don't worry) people will be asked to select a file from their computer, and assuming that its a proper picture file (ends with jpg or png or w.e. the kids use these days) that gets set as their profile pic. If you expect the avatar to be a url, can these files get represented properly? If not, then we gotta rock paper scissors and see who has to make the change lol

Reply:
Sorry for the late reply.

So I think everything depends on how our picture files are being stored. Right now the avatar_url is looking for a string that complies with the @URL hibernate annotation which, by default, uses [this java.net.URL](https://docs.oracle.com/javase/7/docs/api/java/net/URL.html) to validate the string. If you check the ['Types of URL'](http://web.archive.org/web/20051219043731/http://archive.ncsa.uiuc.edu/SDG/Software/Mosaic/Demo/url-primer.html) link, there are additional URL formats it accepts.

Reply:
So whats the best option for saving our picture files?